lib/diagnostic.js

```diff
@@ -6,26 +6,49 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function () {
-
+const normalizeWords = function getNormalizedWords(text) {
```

Comment:
your naming both these first two functions twice which is screwing you up below instead of ```const normalizeWords = function getNormalizedWords(text)``` it should be ```const normalizeWords = function(text)```